tests/test-definitions.sh

```diff
@@ -210,6 +215,36 @@ function verify_ipoib_status {
     check_exit_code "IPoIB is working" "IPoIB is not working!"
 }
 
+function verify_lustre_installation {
+    # Verify lustre client package installation
+    case ${ID} in
+        ubuntu) dpkg -l | grep lustre-client;;
+        almalinux) dnf list installed | grep lustre-client;;
+        * ) ;;
+    esac
+    check_exit_code "Lustre Installed" "Lustre not installed!"
+}
+
+function verify_gdrcopy_installation {
+    # Verify GDRCopy package installation
+    case ${ID} in
+        ubuntu) dpkg -l | grep gdrcopy;;
+        almalinux) dnf list installed | grep gdrcopy;;
+        * ) ;;
+    esac
+    check_exit_code "GDRCopy Installed" "GDRCopy not installed!"
+}
+
+function verify_pssh_installation {
+    # Verify PSSH package installation
+    case ${ID} in
+        ubuntu) dpkg -l | grep pssh;;
+        almalinux) dnf list installed | grep pssh;;
```

Comment:
I have seen pssh appear under different name such as `parallel-ssh` is the package name parallel ssh `pssh` across all distros?

Reply:
on both Alma and Ubuntu we install a package named "pssh". 

We seem to use a mix of parallel ssh packages on our validation pipeline and I don't know why.